pkg/apis/garden/v1beta1/types.go

```diff
@@ -472,9 +472,16 @@ type SeedSpec struct {
 
 // SeedStatus holds the most recently observed status of the Seed cluster.
 type SeedStatus struct {
+	// Gardener holds information about the Gardener which last acted on the Shoot.
+	// +optional
+	Gardener Gardener `json:"gardener,omitempty"`
 	// Conditions represents the latest available observations of a Seed's current state.
 	// +optional
 	Conditions []gardencorev1alpha1.Condition `json:"conditions,omitempty"`
+	// ObservedGeneration is the most recent generation observed for this Shoot. It corresponds to the
```

Comment:
Should be ObservedGeneration is the most recent generation observed for this **Seed**.?

Reply:
I'm not sure whether I get your question correct. If you're asking whether they are the same: They don't have to be the same, the seed controller updates this as soon as it reconciles a Seed. Since we're introducing this field in this version, every Seed will begin in an 'unhealthy' state due to no up to date `ObservedGeneration` but as soon as it's reconciled, it will be ok again.

Reply:
Its only related to the comment above the ObservedGeneration. I think instead of "Shoot" it should be "Seed" because we are in the SeedStatus.

Reply:
Ahh, good catch! Updated it.